packages/blockchain-wallet-v4-frontend/src/modals/BuySell/ThreeDSHandlerStripe/index.tsx

```diff
@@ -12,6 +12,7 @@ import { getData } from './selectors'
 import Success from './template.success'
 
 const ThreeDSHandlerStripe = ({ data, ...props }: Props) => {
+  // TODO pass the logic to here
```

Comment:
is this left for later or? do we have follow up ticket for this?

Reply:
I'll work on it right now